cfpb/cfgov-refresh
# BAH: Handle existing journey URLs with `/process/` segment

We're working on migrating the journey pages from the OAH repo into Wagtail. The pages currently have the URL structure `/owning-a-home/process/{step name}/`, but we'd prefer not to create a `process` page in the Wagtail tree since there's no unique content for it (`/process/` shows the same content as `/process/prepare/`). The migration coincides with a home-buying season campaign, though, and for various SEO/analytics reasons, we've been asked not to change the journey page URLs or add redirects at this time.

This PR adds temporary hacks to:
- load the Wagtail versions of the journey pages at their existing `/process/` URLs
- add a `/process/` segment to the journey side nav URLs
- add `Buying a House` link to breadcrumbs of its child pages & `/process/` segment in breadcrumb journey URLs


## Additions

- Serve journey pages at `/process/` URLs
- Add workaround for Journey page side nav URLs

## Testing

1. Check out branch
2. Create /explore and /prepare pages under Buying a House, and a /explore-child page under /explore
3. Verify that Wagtail journey pages load at  `/process/` URLs: http://localhost:8000/owning-a-home/process/prepare, http://localhost:8000/owning-a-home/process/explore, http://localhost:8000/owning-a-home/process/explore/explore-child
4. Verify that journey page side nav link URLs include `/process/` segment
5. Verify that `Buying a House` shows up in breadcrumbs and breadcrumb URLs include `/process/` segment

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
